CONTRIBUTING.md

```diff
@@ -64,6 +61,28 @@ and notifies you via email and through status badges in pull requests
 about problems.
 
 
+## Debugging
+
+To see the output of the Git commands run in tests, you can either set the
+`DEBUG_COMMANDS` environment variable while running your specs:
+
+```bash
+$ DEBUG_COMMANDS=true cucumber <filename>[:<lineno>]
+```
+
+Alternatively, you can also add a `@debug-commands` flag to the respective
+Cucumber spec:
+
+  ```cucumber
+  @debug-commands
+  Scenario: foo bar baz
+    Given ...
+  ```
+
+For even more detailed output, you can use the `DEBUG` variable or tag
+in a similar fashion.
```

Comment:
Let explain what DEBUG actually does - prints every shell command executed and its output.